source/ndr.rst

```diff
@@ -238,9 +238,10 @@ You may choose to run the model with either Nitrogen or Phosphorus or both at th
   Example biophysical table (only to be used as an example, your LULC classes and corresponding values will be different):
 
   .. csv-table::
-    :file: ndr/ndr_biophysical_table_example.csv
+    :file: ../invest-sample-data/NDR/biophysical_table_gura.csv

```

Comment:
Do you think that showing a table with extra columns in this case is a little confusing? I also notice that the sample data uses `eff_p` as opposed to `eff_n`. I feel like in this case the sample data might not be robust to helping explain the capabilities of the model? 

I also noticed that some of the descriptions above mention things like:
>For example, high values (0.6 to 0.8) may be assigned to all natural vegetation types (such as forests, natural pastures, wetlands, or prairie)

These don't necessarily jive (jibe?) with what we have in the sample data.